public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderMenuProvider.tsx

```diff
@@ -17,7 +17,7 @@ interface Props {
   children: (props: PanelHeaderMenuProviderApi) => ReactElement;
 }
 
-export const PanelHeaderMenuProvider: FC<Props> = ({ panel, dashboard, children }) => {
+export function PanelHeaderMenuProvider({ panel, dashboard, children }: Props) {
```

Comment:
This provider component look like it can just be rewritten to a hook does not need to be a component using children render prop

Reply:
Yes, let's do it in a different PR